langchain/llms/manifest.py

```diff
@@ -0,0 +1,35 @@
+from langchain.llms.base import LLM
+from typing import Any, Dict, Optional, List
+from pydantic import BaseModel, Extra, root_validator
+
+
+class ManifestWrapper(BaseModel, LLM):
+
+    client: Any  #: :meta private:
+    llm_kwargs: Optional[Dict] = None
+
+    class Config:
+        """Configuration for this pydantic object."""
+
+        extra = Extra.forbid
+
+    @root_validator()
+    def validate_environment(cls, values: Dict) -> Dict:
+        """Validate that python package exists in environment."""
+        try:
+            from manifest import Manifest
+
+            if not isinstance(values["client"], Manifest):
+                raise ValueError
+        except ImportError:
+            raise ValueError(
+                "Could not import manifest python package. "
+                "Please it install it with `pip install manifest`."
+            )
+        return values
+
+    def __call__(self, prompt: str, stop: Optional[List[str]] = None) -> str:
+        if stop is not None:
+            raise NotImplementedError("Need to check how to do this")
```

Comment:
I see. Ya so HF (last time I checked) didn't easily support stop sequences. So one way around this is something like

```
response = manifest.run(prompt, **kwargs, return_response=True)
if stop:
    # Assuming the goal is to get the first stop token
    shortest_ans = sorted([response.get_response(st) for st in stop], key=len)
    ans = shortest_ans[0]
else:
    ans = response.get_response()
return ans
```